### DISCUSSION TITLE
Use useFieldArray to get an array of objects

### DISCUSSION THREAD

**Discussion by tuddev** [Mini-ghost/vorms#45] [General]
Hello! Can you tell what is the best way to use useFieldArray to get an array of objects from form?

**Comment by Mini-ghost** [comment]
Hey there! If you're looking to work with an array of objects in a form using `useFieldArray`, I've created a simple example to help you get started. This can serve as a starting point, and if you have any specific requirements or questions, feel free to ask for more guidance.

You can check out the example: https://stackblitz.com/edit/vorms-field-array-t1g5ns?file=src%2FApp.vue.

Happy coding!

**Reply by tuddev** [reply]
Hello! Thank you for your answer. You have created a wonderful tool!
